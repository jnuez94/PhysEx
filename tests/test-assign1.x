int a;
int b;
int c;
int d;
int e;
int f;

void func simulation () {
	a = 12;
	b = 5;
	c = 6;
	d = 52;
	e = 3;
	f = 2;

	printi(a);
	printi(b);
	printi(c);
	printi(d);
	printi(e);
	printi(f);
}
