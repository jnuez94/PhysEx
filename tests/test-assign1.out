12
5
6
52
3
2